packages/react/src/components/Toolbar/ToolbarItem.tsx

```diff
@@ -101,6 +105,20 @@ class ToolbarItem extends UIComponent<WithAsProp<ToolbarItemProps>, ToolbarItemS
         {childrenExist(children) ? children : Icon.create(icon)}
       </ElementType>
     )
+
+    if (popup) {
+      return Popup.create(popup, {
+        defaultProps: {
+          accessibility: popupFocusTrapBehavior,
+        },
+        overrideProps: {
+          trigger: renderedItem,
+          children: undefined,
```

Comment:
this is a bit unexpected, why is it needed? conflict with `trigger` prop that cannot be defined at the same time?

Reply:
Are we trying to override this if something is specified?

Reply:
Yup, `children` collides with `trigger`, so we need to force-reset them.
Changed typings for `popup` prop to make it clearer and added comment.